src/tests/multihost/alltests/test_kcm.py

```diff
@@ -93,34 +90,27 @@ def test_refresh_contain_timestamp(self,
                                                    f"{log_location}")
         assert 'modifyTimestamp>=' not in grep_cmd.stdout_text
 
-    @pytest.mark.no_tier
+    @pytest.mark.tier1_2
     def test_kcm_check_socket_path(self, multihost, backupsssdconf):
         """
         :title: kcm: Test socket path when sssd-kcm is activated by systemd
         :id: 6425bf2c-d07e-4d65-b15d-946141422f96
         :ticket: https://github.com/SSSD/sssd/issues/5406
         """
-        # Start from a known-good state after removing log file and adding a
-        # new socket path
+        client = sssdTools(multihost.client[0])
         domain_log = '/var/log/sssd/sssd_kcm.log'
         multihost.client[0].service_sssd('stop')
-        multihost.client[0].run_command("systemctl stop sssd-kcm")
-        client = sssdTools(multihost.client[0])
-        client.remove_sss_cache(domain_log)
```

Comment:
clearing /var/log/sssd/sssd_kcm.log is missing. Can be added on line 104

Reply:
We don't require it, as after adding debg_level under kcm section and restarting sssd-kcm, sssd_kcm.log file generate.

Reply:
The log file is generated in the previous tests and would have their logs right?

Reply:
I agree with Steeve that the logs need to be cleared. IIRC, there was some artifact from previous tests that would make this test pass when it shouldn't.

Reply:
no, in the first test case we are generating the log but backupsssdconf restore the sssd and we don't have kcm section at end of first test case.
In the second test case, we are not adding kcm section so no kcm log generated in second test case also.
But if you think, we can add 
tools.clear_sssd_cache() but if we add tools.remove_sss_cache('/var/log/sssd/sssd_kcm.log') test failing with no file found.

Reply:
@sgoveas @ikerexxe you are right.
but if we add client.remove_sss_cache(domain_log) then while running individual test case it might fail due no file found,
so adding tools.clear_sssd_cache().

Reply:
@madhuriupadhye resolved this by updating remove_sssd_cache function #5656